multichain-testing/test/cosmos-staking.test.ts

```diff
@@ -0,0 +1,94 @@
+import anyTest from '@endo/ses-ava/prepare-endo.js';
+import type { TestFn } from 'ava';
+import { generateMnemonic } from '../tools/wallet.js';
+import { makeQueryClient } from '../tools/query.js';
+import { commonSetup } from './support.js';
+// import { makeDoOffer } from '../tools/e2e-tools.js';
+
+const test = anyTest as TestFn<Awaited<ReturnType<typeof commonSetup>>>;
+
+const KEYS = ['user1'];
+
+const deleteKeys = async (deleteKey: (name: string) => Promise<string>) => {
+  for (const key of KEYS) {
+    try {
+      await deleteKey(key);
+    } catch (_e) {
+      // ignore
+    }
+  }
+};
+
+test.before(async (t) => {
+  t.context = await commonSetup(t);
+  const { deleteKey } = t.context;
+  await deleteKeys(deleteKey);
```

Comment:
this reads oddly. consider,
```suggestion
  await deleteTestKeys(deleteKey);
```
or inlining,
```js
  await Promise.all(KEYS.map(key => deleteKey(key).catch()));
```